child_compassion/model/child_compassion.py

```diff
@@ -276,8 +276,13 @@ def get_infos(self, cr, uid, ids, context=None):
         proj_obj = self.pool.get('compassion.project')
         res = True
         for child in self.browse(cr, uid, ids, context):
-            res = res and self._get_case_study(cr, uid, child, context)
-            self._get_basic_informations(cr, uid, child.id)
+            if child.type != 'LDP':

```

Comment:
Suggestion:
```python
if child.type == 'LDP':
```
and invert test. To me (and it's very subjective), it's easier to read a conditional test rather than the "inverse-test". But, once more, this is only a style suggestion.